app/models/accounting/quickbooks/updater.rb

```diff
@@ -1,23 +1,22 @@
 module Accounting
   module Quickbooks
-    class FullSyncRequiredError < StandardError; end
-    class NotConnectedError < StandardError; end
 
     # Reponsible for grabbing only the updates that have happened in quickbooks
     # since the last time this class was run. If no quickbooks data exists in the sytem
     # yet, FullFetcher will need to be run first.
     class Updater
       attr_reader :qb_connection
 
+      MIN_TIME_BETWEEN_UPDATES = 5.seconds
```

Comment:
I thought this would be 15 seconds

Reply:
Yes, I reconsidered. There is a comment about this elsewhere in the file I think.